src/dvpio/read/shapes/geometry.py

```diff
@@ -1,9 +1,15 @@
+from typing import Literal
+
 import numpy as np
 from numpy.typing import NDArray
+from skimage.transform import estimate_transform
 
 
 def compute_affine_transformation(
```

Comment:
probably this function name should be updated if different transforms can be calculated based on passed params